web_development_101/installations/project_your_first_rails_app.md

```diff
@@ -2,6 +2,15 @@
 
 With Ruby installed, you're all set to create your first Ruby on Rails application!
 
+Before continuing, let's review a few best practices to keep in mind:
+
+* Follow the directions closely, and don't skip over any sections.
+* **Do NOT use `sudo` unless The Odin Project specifically says to do so.** In addition, never run as the `root` user. Failing to follow this advice can cause *a lot* of headaches. In some instances, you might see a message in the terminal telling you to use `sudo` and/or to install something with `apt`. **Ignore what the terminal says** and follow the instructions below.
```

Comment:
> ever run as the `root` user

Is there a chance the student won't know what the root user is?

Reply:
It's very likely that they won't know what that means. Should I just delete that part or say something like "if you know what this means, don't do it"?

Reply:
I think most people will be signed in to a user account, and will only be able to run commands as the root user by using `sudo`. We could always try putting this out without any mention of root user and seeing if anyone runs into a problem like this.

Reply:
Alright, I removed that bit about the root user.